annoyed-alligators/socl_media/apps/feed/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+
+class Posts(models.Model):
```

Comment:
Change the model name to `Post` instead of `Posts`. This is because this is basically a model which tells about the attributes of one single post. And it's better to change the name early, before it's used in many places.

Reply:
okay will do